ota-plus-web/app/reactapp/src/stores/ImpactAnalysisStore.js

```diff
@@ -2,7 +2,6 @@
 
 import { observable } from 'mobx';
 import axios from 'axios';
-import { API_IMPACT_ANALYSIS_FETCH } from '../config';
 import { resetAsync, handleAsyncSuccess, handleAsyncError } from '../utils/Common';
```

Comment:
handleAsyncSuccess and handleAsyncError are not needed anymore

Reply:
Removed.